cmd/kube-proxy/app/server_test.go

```diff
@@ -392,6 +392,10 @@ func TestProcessHostnameOverrideFlag(t *testing.T) {
 			hostnameOverrideFlag: "  bar ",
 			expectedHostname:     "bar",
 		},
+		{
+			name:                 "Hostname is space",
```

Comment:
why is this tested needed/useful?

Reply:
because the testcase didn't cover that:
![image](https://user-images.githubusercontent.com/20693362/51421946-cfd5d600-1be1-11e9-878a-519193e2afcb.png)